templates/task.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   volumes:
     - name: data-dir
       hostPath:
-        path: /var/csi/rawfile
+        path: {rawfile_dir}
```

Comment:
Actually, this needs to be passed into the templating function so that it can get rendered.

Reply:
![image](https://user-images.githubusercontent.com/26027006/95216728-9e20fa80-081c-11eb-97fc-7bb58125e390.png)
can i change it into templating function like this?

Reply:
The values are interpolated not by helm, but the python code. So that's where the code needs to be updated.